### ISSUE TITLE
documentation mentions a non-existing option  --unitslat=us

### ISSUE THREAD

**Issue by alevchuk** [brendangregg/HeatMap#7]
It seems that 
http://www.brendangregg.com/perf.html#HeatMaps
is out of date:

```
./trace2heatmap.pl --unitstime=us --unitslat=us --maxlat=50000 out.lat_us > out.svg
```
Produces:
> Unknown option: unitslat

**Comment by martinvonwittich** [comment]
Yeah, the option `--unitslatency` (which is what `--unitslat` actually meant, Perl's `Getopt::Long` allows abbreviated option names as long as they're unique) was renamed to `--unitslabel` in 462b71d58734f1c3883527d385702efc43fd0625.
